lindera/src/tokenizer.rs

```diff
@@ -553,31 +531,6 @@ impl Tokenizer {
 
         Ok(tokens)
     }
-
-    /// Tokenize the text
-    ///
-    /// # Arguments
-    ///
-    /// * `text`: Japanese text
-    ///
-    /// returns: Result<Vec<&str>, LinderaError>
-    ///
-    /// * Vec<&str> : the list of `&str` if succeeded
-    /// * LinderaError : Error message with LinderaErrorKind
-    ///
-    pub fn tokenize_str<'a>(&self, mut text: &'a str) -> LinderaResult<Vec<&'a str>> {
-        let mut lattice = Lattice::default();
-        let mut tokens = Vec::new();
-        while let Some(split_idx) = text.find(|c| c == '。' || c == '、') {
```

Comment:
Sorry, GitHub doesn't let me comment on the upper `tokenize` method but the code is the same.
I suggest using the [split_inclusive method on `&str`](https://doc.rust-lang.org/stable/std/primitive.str.html#method.split_inclusive) to do the job:

```rust
    pub fn tokenize<'a>(&self, text: &'a str) -> LinderaResult<Vec<Token<'a>>> {
        let mut lattice = Lattice::default();
        let mut tokens = Vec::new();
        for sub_str in text.split_inclusive(&['。',  '、']) {
            self.tokenize_without_split(sub_str, &mut tokens, &mut lattice)?;
        }

        Ok(tokens)
    }
```

What do you think? 🤔 

Thank you again @mosuka for this improvement!

By the way, this is the [Hacktoberfest](https://hacktoberfest.com/) soon!

Reply:
Wow, I didn't know it. I'll fix it later.
Thanks!

Reply:
@ManyTheFish 
Wow, I didn't know it. I'll fix it later.
Thanks!